### PR TITLE
Added missing CZ strings and made little revision

### PR DESCRIPTION
**Pull request by Kllrt**
Nuff said.

### CODE REVIEW

**Comment by KoffeinFlummi** [comment]
Thanks for your contribution! Please add yourself to the AUTHORS.txt file with a valid email address before we merge this.

**Comment by Kllrt** [comment]
Sure

File: addons/missileguidance/stringtable.xml [hunk 1]
```diff
@@ -28,80 +28,8 @@
...
-      <French></French>
-      <Polish></Polish>
-      <German></German>
-      <Czech></Czech>
-      <Italian></Italian>
-      <Portuguese></Portuguese>
-      <Hungarian></Hungarian>
-      <Russian></Russian>
-    </Key>
+    <Key ID="STR_ACE_Hydra70_DAGR"><English>Hydra-70 DAGR Missile</English><Spanish></Spanish><French></French><Polish></Polish><German></German><Czech>Hydra-70 DAGR</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hydra70_DAGR_Short"><English>DAGR</English><Spanish></Spanish><French></French><Polish></Polish><German></German><Czech>DAGR</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hydra70_DAGR_Desc"><English>Hydra-70 DAGR Laser Guided Missile</English><Spanish></Spanish><French></French><Polish></Polish><German></German><Czech>Hydra-70 DAGR Laserově naváděna střela</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key>
```
Comment (KoffeinFlummi) [inline addons/missileguidance/stringtable.xml]: Something broke here ...

File: addons/missileguidance/stringtable.xml [hunk 2]
```diff
@@ -28,80 +28,8 @@
...
-      <German></German>
-      <Czech></Czech>
-      <Italian></Italian>
-      <Portuguese></Portuguese>
-      <Hungarian></Hungarian>
-      <Russian></Russian>
-    </Key>
-    
-  </Package>
+    <Key ID="STR_ACE_Hellfire_AGM114K"><English>Hellfire II AGM-114K Missile</English><Spanish></Spanish><French></French><Polish></Polish><German></German><Czech>Hellfire II AGM-114K</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hellfire_AGM114K_Short"><English>AGM-114K</English><Spanish></Spanish><French></French><Polish></Polish><German></German><Czech>AGM-114K</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hellfire_AGM114K_desc"><English>Hellfire II AGM-114K Laser Guided Missile</English><Spanish></Spanish><French></French><Polish></Polish><German></German><Czech>Hellfire II AGM-114K Laserově naváděná střela</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key></Package>
```
Comment (KoffeinFlummi) [inline addons/missileguidance/stringtable.xml]: Also here ...

File: addons/missileguidance/stringtable.xml (hunk 1 above)
Comment (Kllrt) [inline addons/missileguidance/stringtable.xml]: I see, but I was using Tabler to make that translation.

**Comment by Kllrt** [comment]
Ok, so I fixed it manually.

**Comment by Kllrt** [comment]
Shit thiss merging is a mess. Can someone help me with it? :(

**Comment by KoffeinFlummi** [comment]
Thanks.

**Pull request merged**